<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>log_Manage</class>
 <widget class="QMainWindow" name="log_Manage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="tableView">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_showall">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>51</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>显示全表</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_out_2">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>51</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>导出Excel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_name">
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>51</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="placeholderText">
           <string>请输入登陆者姓名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_IP">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>51</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="placeholderText">
           <string>请输入IP：xxx.xxx.xxx.xxx</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_play">
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>51</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="placeholderText">
           <string>请输入登陆角色</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_serch">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>51</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
